content/docs/gto/get-started.md

```diff
@@ -0,0 +1,162 @@
+
+Assuming GTO is already [installed](/doc/gto/install) in your active python
+environment, let's clone the example repo
+
+```cli
+$ git clone https://github.com/iterative/example-gto
+$ cd example-gto
+```
+
+This repo represents a simple example of Machine Learning Model Registry. Let's
+review it:
+
+```cli
+$ gto show
+╒══════════╤══════════╤════════╤═════════╤════════════╕
+│ name     │ latest   │ #dev   │ #prod   │ #staging   │
+╞══════════╪══════════╪════════╪═════════╪════════════╡
+│ churn    │ v3.1.1   │ v3.1.1 │ v3.0.0  │ v3.1.0     │
+│ segment  │ v0.4.1   │ v0.4.1 │ -       │ -          │
+│ cv-class │ v0.1.13  │ -      │ -       │ -          │
+╘══════════╧══════════╧════════╧═════════╧════════════╛
+```
+
+Here we have 3 models: `churn`, `segment` and `cv-class`. The latest versions of
+them are shown in the column named `latest`. The latest is selected as the one
+having the greatest [SemVer](https://semver.org).
+
+Model versions could be promoted to different stages. Here we have 3 of them:
+`dev`, `prod` and `staging`. When a model was never promoted to a stage, we see
+`-` in the field.
+
+## Registering versions and assigning stages
+
+GTO can [register version](/doc/gto/command-reference/register) of artifacts and
+[assign them to stages](/doc/gto/command-reference/assign). Both functionalities
+work in a similar way, so let's walkthough only one of them here.
+
+Let's suppose the version `v0.1.13` of `cv-class` was looking very promising,
+and now we want to promote it to `dev` to test it:
+
+```cli
+$ gto assign cv-class --version v0.1.13 --stage dev
+Created git tag 'cv-class#dev#1' that assigns stage to version 'v0.1.13'
+To push the changes upstream, run:
+    git push origin cv-class#dev#1
+```
+
+We can see, that GTO created a Git tag with a special format that contains
+instruction to assign a stage to a version. We can push to Git repository to
+start the CI, but let's ensure that changed our Registry first.
+
+```cli
+$ gto show
+╒══════════╤══════════╤═════════╤═════════╤════════════╕
+│ name     │ latest   │ #dev    │ #prod   │ #staging   │
+╞══════════╪══════════╪═════════╪═════════╪════════════╡
+│ churn    │ v3.1.1   │ v3.1.1  │ v3.0.0  │ v3.1.1     │
+│ segment  │ v0.4.1   │ v0.4.1  │ -       │ -          │
+│ cv-class │ v0.1.13  │ v0.1.13 │ -       │ -          │
+│ awesome  │ v0.0.1   │ -       │ -       │ -          │
+╘══════════╧══════════╧═════════╧═════════╧════════════╛
+```
+
+As we can see, the version `v0.1.13` of `cv-class` was promoted to `dev` stage.
+
+## Acting downstream
+
+The power of using Git tags to register versions and assign stages is simple: we
+can act upon them in well-known way - in CI/CD.
+
+To see how it works, let's fork the
+[example-gto repo](https://github.com/iterative/example-gto) and push the tag we
+just created to GitHub. For CI/CD to start, you'll need to enable them on the
+"Actions" page of your fork.
+
+<details>
+
+### Step-by-step instruction
+
+will be here
```

Comment:
Would it be easy to setup something like a [post-receive](https://git-scm.com/docs/githooks#post-receive) hook in a local Git remote (bare repo?) to demo this? May be asier to reproduce without leaving the OS, but may also be too obscure...

Reply:
Good idea, but I want to showcase a real CI job here. IIUC, we can't showcase CI job or anything similar without fork. WDYT?

Reply:
Makes sense. Maybe we can hide the forking instructions and other `<details>` like that though?

Reply:
Not sure I got you. We already hide it under `<details>`, right? Do you suggest to remove that at all?

Reply:
Yeah I see that now. I think the whole sentence about forking would be hidden but it's not major...